nano/templates/antiquems.tmpl

```diff
@@ -0,0 +1,42 @@
+<div class="item">
+	<div class="itemLabel">
+		Stored Capacity:
+	</div>
+	<div class="itemContent">
+		{{:helper.displayBar(data.charge, 0, 2000, data.active ? 'good' : 'average')}}
+		<div class="statusValue">
+			{{:helper.round(data.charge)}} megajoules
+		</div>
+	</div>
+	<div class="itemLabel">
+		Power Management:
+	</div>
+	<div class="itemContent">
+		{{:helper.link('Toggle Power', 'power', {'toggle' : 1})}}
+		{{:helper.link('Set Consumption', 'battery-3', {'set_draw' : 1})}}
+	</div>
+	<div class="itemLabel">
+		Consumption:
+	</div>
+	<div class="itemContent">
+		{{:data.consumption}} megawatts 
```

Comment:
round this

Reply:
it does get rounded before it gets passed at all

Reply:
oh? I could swear I got weird decimals when testing this

Reply:
stored amount doesn't round, maybe there?